.github/workflows/ci.yml

```
@@ -79,3 +79,113 @@ jobs:
     steps:
       - name: Run Codecov 
         run: bash <(curl -s https://codecov.io/bash)
+
+  # Increments to the next tag on master only, bumps version based on the recent commits
+  new_tag:
+    if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' && github.event_name == 'push'
```

Maybe add brackets around the or?
```suggestion
    if: (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master') && github.event_name == 'push'
```